app/models/user.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable func-names */
+/* eslint-disable no-empty-function */
```

Comment:
This rules are needed for this [line](https://github.com/Wolox/express-js-bootstrap/pull/96/files#diff-a4b3038d4ee7f4db64a479a6b17460a6R38) ? i think then could be removed in that case, if is not the case and is general is a little tricky use this set of rules in each model

Reply:
@mpizzagalli what do you think?